src/hooks/useAddTokenToMetamask.ts

```diff
@@ -30,7 +30,10 @@ export default function useAddTokenToMetamask(token: Token | undefined): {
               address: token.addresses[chainId],
               symbol: token.symbol,
               decimals: token.decimals,
-              image: getTokenIconPath(token.symbol),
+              image:
+                token.symbol === "SDL"
+                  ? "https://assets.coingecko.com/coins/images/20476/small/edit_saddle.png?1660734429"
+                  : getTokenIconPath(token.symbol),
```

Comment:
I think it would be better if we just fix this at the source by making that helper function return a url rather than relative path, eg:

```typescript
export function getTokenIconPath(tokenSymbol: string): string {
  const iconName = tokenSymbol.toLowerCase().includes("saddle")
    ? "saddle_lp_token"
    : tokenSymbol.toLowerCase()
  const x = new Image()
  x.src = `/static/icons/svg/${iconName}.svg`

  return x.src
}
```

Alternatively, I think just doing `${window.location.host}/static/icons/svg/${iconName}.svg` should work as well

Reply:
Actually, gonna +1 on this comment.  This is a better solution.

Reply:
🐐